modules/virt-about-cloning.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-creating-vms-by-cloning-pvcs.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="virt-about-cloning_{context}"]
+= About cloning
+
+When cloning a data volume, the Containerized Data Importer (CDI) chooses one of these Container Storage Interface (CSI) clone methods:
```

Comment:
Only two of the methods are CSI: CSI volume cloning, and Smart cloning (ie. CSI Snapshots).  Host-assisted is a generic copy method that can be used when a CSI option is not available.